src/contracts/version/transactions/registerExchange.ts

```diff
@@ -0,0 +1,39 @@
+import web3EthAbi from 'web3-eth-abi';
+import { Address } from '@melonproject/token-math/address';
+import {
+  transactionFactory,
+  PrepareArgsFunction,
+  EnhancedExecute,
+} from '~/utils/solidity/transactionFactory';
+import { Contracts } from '~/Contracts';
+import { FunctionSignatures } from '~/contracts/fund/trading/utils/FunctionSignatures';
+interface RegisterExchangeArgs {
+  exchange: Address;
```

Comment:
Similar to [above comment](https://github.com/melonproject/protocol/pull/600/files#r239409022), we may be able to pass in `ExchangeConfig` instances similar to how we do in `createComponent` to achieve some code reuse.